tests/toranj/openthread-core-toranj-config.h

```diff
@@ -67,6 +67,14 @@
  */
 #define OPENTHREAD_CONFIG_BORDER_ROUTER_ENABLE 1
 
+/**
+ * @def OPENTHREAD_CONFIG_BORDER_ROUTING_ENABLE
+ *
+ * Define to 1 to enable Border Routing support.
+ *
+ */
+#define OPENTHREAD_CONFIG_BORDER_ROUTING_ENABLE 1
```

Comment:
Is this change related to this PR?

Reply:
No. Not directly related, but enabling it under `toranj` does help cover the `border_router` src codes under more CI builds and should be harmless since on `toranj` tests it is not enabled.